src/advanced/bitcoin-anchoring.md

```diff
@@ -209,13 +215,22 @@ secured.
 
 #### Synchronization with Bitcoin Blockchain
 
-A separate [method][handler] performs regular synchronization of the list of
-Exonum anchoring transactions with those committed to the Bitcoin Blockchain. It
-takes the latest
-anchoring transaction and checks whether it is present in Bitcoin. If not, the
-handler checks the previous anchoring transactions one by one in the same manner
-until it finds the last anchoring transaction committed to Bitcoin Blockchain.
-The handler then pushes all the uncommitted anchoring transactions to Bitcoin.
+A separate [utility][btc_anchoring_sync] periodically performs
+two actions:
+
+- Creation of a signature for a new anchoring transaction. The utility takes the
+  actual anchoring transaction proposal by the node private API, signs this
+  proposal by the corresponding Bitcoin key and sends this signature back to the
+  validator node. Validator node creates a new vote transaction from this signature
+  and broadcasts it to the other nodes.
+
```

Comment:
Isn't empty line between list elements breaks the list?

Reply:
No, list looks good in preview.